### ISSUE TITLE
Staples_2019: New data submission

### ISSUE THREAD

**Issue by dfalster** [traitecoevo/austraits.build#201]
New dataset to import from Tim Staples:

**Source**: Staples et al (2019). Productivity does not correlate with species and functional diversity in Australian reforestation plantings across a wide climate gradient. Global Ecology and Biogeography. 10.1111/geb.12962
**Custodian**: Timothy Staples
**Handled by:** Caitlan Baxter

Data is available at https://github.com/TimothyStaples/diversity-productivity-reforestation

Note, the trait data include collections from TRY Plant Trait Database, field collection made by the authors of this paper, field collections made from colleagues, and Australian herbarium. We want to filter down to only use new data collected by Tim and colleagues.

Corresponding publication is http://doi.org/10.1111/geb.12962

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )
